x-pack/plugins/security_solution/public/management/pages/policy/view/protection_updates/protection_updates_layout.tsx

```diff
@@ -89,7 +89,7 @@ export const ProtectionUpdatesLayout = React.memo<ProtectionUpdatesLayoutProps>(
     const internalDateFormat = 'YYYY-MM-DD';
     const displayDateFormat = 'MMMM DD, YYYY';
     const formattedDate = moment(deployedVersion, internalDateFormat).format(displayDateFormat);
-    const cutoffDate = moment().subtract(18, 'months'); // Earliest selectable date
+    const cutoffDate = moment().subtract(18, 'months').add(1, 'day'); // Earliest selectable date
```

Comment:
Got curious, why is this needed?

Reply:
Cutoff date API side is 18 months from today, we cant let user select 18 months for it will fail api validation.